src/Api/PubnubApi/Model/Consumer/UserId.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace PubnubApi
+{
+    public class UserId
+    {
+        private string _userId;
+        public UserId(string value)
+        {
+            _userId = value;
+        }
+
+        public override string ToString()
```

Comment:
I'd add an implicit cast to string, it would make the code a little bit cleaner (possible to remove all the .ToString() calls)

Reply:
Added implicit operators.